src/main/java/org/generationcp/middleware/dao/ProjectUserInfoDAO.java

```diff
@@ -42,195 +27,36 @@
-
-	@SuppressWarnings("unchecked")
-	public List<Project> getProjectsByUser(WorkbenchUser user) {
-		try {
-			if (user != null) {
-				Criteria criteria = this.getSession().createCriteria(ProjectUserInfo.class);
-				criteria.add(Restrictions.eq("user.userid", user.getUserid()));
-				criteria.setProjection(Projections.distinct(Projections.property("project")));
-				return criteria.list();
-			}
-		} catch (HibernateException e) {
-			throw new MiddlewareQueryException("Error in getProjectsByUser(user=" + user + ") query from ProjectUserInfoDao: " + e.getMessage(), e);
-		}
-		return new ArrayList<>();
-	}
-
-	@SuppressWarnings("unchecked")
-	public List<WorkbenchUser> getUsersByProjectId(final Long projectId) {
-		final List<WorkbenchUser> users = new ArrayList<>();
-		try {
-			if (projectId != null) {
-				final SQLQuery query = this.getSession().createSQLQuery(ProjectUserInfoDAO.GET_USERS_BY_PROJECT_ID);
-				query.setParameter("projectId", projectId);
-				final List<Object> results = query.list();
-				for (final Object o : results) {
-					final Object[] user = (Object[]) o;
-					final Integer userId = (Integer) user[0];
-					final Integer instalId = (Integer) user[1];
-					final Integer uStatus = (Integer) user[2];
-					final Integer uAccess = (Integer) user[3];
-					final Integer uType = (Integer) user[4];
-					final String uName = (String) user[5];
-					final String upswd = (String) user[6];
-					final Integer personId = (Integer) user[7];
-					final Integer aDate = (Integer) user[8];
-					final Integer cDate = (Integer) user[9];
-					final Person person = new Person();
-					person.setId(personId);
-					person.setFirstName((String) user[10]);
-					person.setMiddleName((String) user[11]);
-					person.setLastName((String) user[12]);
-					final WorkbenchUser u = new WorkbenchUser(userId, instalId, uStatus, uAccess, uType, uName, upswd, person, aDate, cDate);
-					users.add(u);
-				}
-			}
-		} catch (final HibernateException e) {
-			throw new MiddlewareQueryException("Error in getUsersByProjectId(projectId=" + projectId + ") query from ProjectUserInfoDao: "
-					+ e.getMessage(), e);
-		}
-		return users;
-	}
-
-	public List<Integer> getActiveUserIDsByProjectId(final Long projectId) {
-		final List<Integer> userIDs = new ArrayList<>();
-		try {
-			if (projectId != null) {
-				final SQLQuery query = this.getSession().createSQLQuery(ProjectUserInfoDAO.GET_ACTIVE_USER_IDS_BY_PROJECT_ID);
-				query.setParameter("projectId", projectId);
-				return query.list();
-			}
-		} catch (final HibernateException e) {
-			throw new MiddlewareQueryException("Error in getActiveUserIDsByProjectId(projectId=" + projectId + ") query from ProjectUser: "
-					+ e.getMessage(), e);
-		}
-		return userIDs;
-	}
-
-	public ProjectUserInfo getByProjectIdAndUserId(Long projectId, Integer userId) {
+	//Used to get the last opened date, if there is no row, a new one will be created
```

Comment:
This comment is not applicable in this DAO context:
, if there is no row, a new one will be created